### PR TITLE
Np 5008 queryparam issue

### PR DESCRIPTION
**Pull request by RanjithTom**

### CODE REVIEW

File: search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java [hunk 1]
```diff
@@ -77,6 +82,29 @@ void init() throws IOException {
...
+        var response = GatewayResponse.fromOutputStream(outputStream, SearchResourcesResponse.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
+        var actualPageSize = restHighLevelClientWrapper.getSearchRequest().source().size();
+        assertThat(actualPageSize, equalTo(expectedPageSize));
+    }
+
+    @Test
+    void shouldSentDefaultPageSizeRequestWhenPageSizeNotSubmitted() throws IOException {
+
+        InputStream request = queryWithoutQueryParameters();
```
Comment (axthosarouris) [inline search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java:100]: var

File: search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java [hunk 2]
```diff
@@ -136,13 +164,20 @@ void shouldNotSendQueryAndReturnForbiddenWhenUserDoesNotHaveTheAppropriateAccess
...
     void shouldSendQueryIncludingAllIndicesRelevantToCuratorsWorklist() throws IOException {
         handler.handleRequest(queryWithoutQueryParameters(), outputStream, context);
         var searchRequest = restHighLevelClientWrapper.getSearchRequest();
         var indices = Arrays.stream(searchRequest.indices()).collect(Collectors.toList());
-        assertThat(indices, containsInAnyOrder("messages","doirequests"));
+        assertThat(indices, containsInAnyOrder("messages", "doirequests"));
+    }
+
+    private InputStream createRequestWithPageSize(Integer expectedPageSize) throws JsonProcessingException {
+        return new HandlerRequestBuilder<>(JsonUtils.dtoObjectMapper).withQueryParameters(
```
Comment (axthosarouris) [inline search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java:176]: format, each call on separate line.

**Review by axthosarouris — CHANGES_REQUESTED** [review]

File: search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java (hunk 2 above)
Comment (RanjithTom) [reply search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java]: Updated

File: search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java (hunk 1 above)
Comment (brinxmat) [reply search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java:100]: You can replace the InputStream type with `var`

File: search-resources-api/src/main/java/no/unit/nva/search/SearchAllHandler.java [hunk 3]
```diff
@@ -69,6 +69,10 @@ private static IdentityClient defaultIdentityClient() {
         return new IdentityClientImpl();
     }
 
+    private Integer extractPageSize(RequestInfo requestInfo) {
+        return requestInfo.getQueryParameterOpt(PAGE_SIZE_QUERY_PARAM).map(Integer::valueOf).orElse(DEFAULT_PAGE_SIZE);
+    }
```
Comment (brinxmat) [inline search-resources-api/src/main/java/no/unit/nva/search/SearchAllHandler.java:74]: Format the lines for readability

```
    private Integer extractPageSize(RequestInfo requestInfo) {
        return requestInfo.getQueryParameterOpt(PAGE_SIZE_QUERY_PARAM)
            .map(Integer::valueOf)
            .orElse(DEFAULT_PAGE_SIZE);
    }
```

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by brinxmat — APPROVED** [review]

**Review by axthosarouris — DISMISSED** [review]

**Review by axthosarouris — APPROVED** [review]

**Pull request merged**